src/models/DefaultSettings.ts

```diff
@@ -49,17 +49,24 @@ const DefaultSettingsSchema = z.object({
 
 export type DefaultSettings = z.infer<typeof DefaultSettingsSchema>;
 
-export async function getDefaultSettings(env: Env) {
-  const defaultSetttingsClient = await env.data.clients.get("DEFAULT_CLIENT");
-
-  if (!defaultSetttingsClient) {
-    throw new Error("Failed to load default settings tenant");
-  }
-
+export async function getDefaultSettings(env: Env): Promise<DefaultSettings> {
   try {
+    const defaultSetttingsClient = await env.data.clients.get("DEFAULT_CLIENT");
```

Comment:
This was required before. I think having a parent or default tenant shouldn't be required.